torch/__init__.py

```diff
@@ -345,20 +345,20 @@ def set_default_dtype(d):
     """
     _C._set_default_dtype(d)
 
-def use_deterministic_algorithms(d):
+def use_deterministic_algorithms(mode):
     r""" Sets whether PyTorch operations must use "deterministic"
     algorithms. That is, algorithms which, given the same input, and when
     run on the same software and hardware, always produce the same output.
     When True, operations will use deterministic algorithms when available,
     and if only nondeterministic algorithms are available they will throw a
-    :class:RuntimeError when called.
+    :class:`RuntimeError` when called.
 
     .. warning::
```

Comment:
I suppose we should wait to rip this label off until the feature review committee agrees ;)